### PR TITLE
ENT-4235 Quiet runalerts during normal operation

### PR DESCRIPTION
**Pull request by nickanderson**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thanks for submitting a PR! Maybe @craigcomstock can review this?

File: cfe_internal/enterprise/templates/runalerts.php.mustache [hunk 1]
```diff
@@ -9,11 +9,15 @@ if(PHP_SAPI != 'cli')
 
 while(1 == 1)
 {
-  print("Checking for sql alerts");
+  status=0;
```
Comment (aleksandrychev) [inline cfe_internal/enterprise/templates/runalerts.php.mustache]: all variables should start with $, $status

Comment (nickanderson) [reply cfe_internal/enterprise/templates/runalerts.php.mustache]: Thanks.

Comment (nickanderson) [reply cfe_internal/enterprise/templates/runalerts.php.mustache]: Updated, squashed.

**Pull request merged**